### PR TITLE
Updated the output for dvc version command in docs

### PR DESCRIPTION
**Pull request by sahilbhosale63**
`dvc version` command output update in docs #1635

### CODE REVIEW

File: content/docs/command-reference/dag.md [hunk 1]
```diff
@@ -15,25 +15,26 @@ positional arguments:
...
+
+Using DVC, you can create a metafile `data.dvc` which allows us to reproduce
+each stage of a pipeline using `dvc repro`. At the end of every pipeline, you
+can save your output in a metrics file using `dvc metrics` command. This file
+will help you in comparing the results of every experiment.
+
+DVC provides a `dvc dag` command which creates a direct acyclic graph
+([DAG](https://en.wikipedia.org/wiki/Directed_acyclic_graph)) that gives a
+pictorial view of a pipeline. It also tells you in which stage of a pipeline you
+are currently in.
```
Comment (shcheklein) [inline content/docs/command-reference/dag.md:38]: Unrelated changes ^^ ?

Comment (jorgeorpinel) [reply content/docs/command-reference/dag.md:38]: Yep. @sahilbhosale63 you probably created this branch on top of a branch for another PR. Please either rebase or cherry-pick the version-specific commits into master, or just un-do the changes to this file (dag) e.g. with `git checkout master -- content/docs/command-reference/dag.md` (+add+commit).

**Comment by shcheklein** [comment]
Closing this as stale. This PR also requires some substantial rework. @sahilbhosale63 if you still want to get it done, please ping us and we'll help you.
